sqlalchemy_ibmi/base.py

```diff
@@ -730,68 +754,6 @@ def set_isolation_level(self, dbapi_conn, level):
     def get_isolation_level(self, dbapi_conn):
         pass
 
-    def normalize_name(self, name):
-        return self._reflector.normalize_name(name)
-
-    def denormalize_name(self, name):
-        return self._reflector.denormalize_name(name)
-
-    def _get_default_schema_name(self, connection):
-        return self._reflector._get_default_schema_name(connection)
-
-    def has_table(self, connection, table_name, schema=None):
-        return self._reflector.has_table(connection, table_name, schema=schema)
-
-    def has_sequence(self, connection, sequence_name, schema=None):
-        return self._reflector.has_sequence(connection, sequence_name,
-                                            schema=schema)
-
-    def get_schema_names(self, connection, **kw):
-        return self._reflector.get_schema_names(connection, **kw)
-
-    def get_table_names(self, connection, schema=None, **kw):
-        return self._reflector.get_table_names(connection, schema=schema, **kw)
-
-    def get_view_names(self, connection, schema=None, **kw):
-        return self._reflector.get_view_names(connection, schema=schema, **kw)
-
-    def get_view_definition(self, connection, viewname, schema=None, **kw):
-        return self._reflector.get_view_definition(
-            connection, viewname, schema=schema, **kw)
-
-    def get_columns(self, connection, table_name, schema=None, **kw):
-        return self._reflector.get_columns(
-            connection, table_name, schema=schema, **kw)
-
-    def get_primary_keys(self, connection, table_name, schema=None, **kw):
-        return self._reflector.get_primary_keys(
-            connection, table_name, schema=schema, **kw)
-
-    def get_foreign_keys(self, connection, table_name, schema=None, **kw):
-        return self._reflector.get_foreign_keys(
-            connection, table_name, schema=schema, **kw)
-
-    def get_incoming_foreign_keys(
-            self,
-            connection,
-            table_name,
-            schema=None,
-            **kw):
-        return self._reflector.get_incoming_foreign_keys(
-            connection, table_name, schema=schema, **kw)
```

Comment:
Huh, this seems to have been something unique to ibm_db_sa and only implemented for Db2 LUW.

Reply:
Is there a reason this wouldn't work on IBM i ?
https://github.com/IBM/sqlalchemy-ibmi/commit/842a2c7393971676555949b87c4c81e4a60e4dfc

Reply:
No clue, but I'm not sure _why_ it was added or what it would be used for so I'm inclined to just leave it be gone.